python/pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [project]
 name = "bxsolana-trader-proto"
-version = "0.0.28"
+version = "0.0.28.3"
```

Comment:
just bump this to `0.0.29` I think? we can probably be a bit more aggressive with bumping the minor version stuff TBH

Reply:
yeah, this is just for testing. i was going to bump it to `"0.0.29"` and publish it